modules/installation-registry-osp-creating-custom-pvc.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage.adoc
+
+[id="installation-registry-osp-creating-custom-pvc_{context}"]
+= Configuring an image registry with a custom back end on clusters that run on {rh-openstack}
+
+After you install a cluster on {rh-openstack-first}, you can use a Cinder volume that is in a specific availability zone for registry storage.
+
+.Procedure
+
+. Create a YAML file that specifies a storage class that includes the availability zone that you want to use. For example:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: custom-csi-storageclass
+provisioner: cinder.csi.openstack.org
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  availability: <availability_zone_name>
+----
++
+[NOTE]
+====
+{product-title} does not verify the existence of the availability zone you choose. Verify the name of the availability zone before you apply the configuration.
+====
+
+. From a command line, apply the configuration:
++
+.Example input
```

Comment:
I think you can omit these `.Example input` block headings without harming our customers.

Reply:
```suggestion
```